packages/tablev2/tablev2.test.tsx

```diff
@@ -260,7 +260,9 @@ describe("Table v2", () => {
       );
 
       expect(sorterFn).toHaveBeenCalledWith("asc", 1);
-      component.find("#name button").simulate("click");
+
+      fireEvent.click(screen.getByTestId("table-headercell-name-button"));
```

Comment:
```suggestion
  fireEvent.click(getByTestId("table-headercell-name-button"));
```
Then you will not need screen.